index.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tessy's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+<main>
+  <header>
+    <nav>
+      <div class="logo"><p>John Doe</p></div>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>   
+  </header>
+    <section class="headline" id="headline">
+      <div class="main-box">
+        <h1 class="main">
+        Hi There.<br>
+        I'm Tessy
+        </h1>
+        <sub-title class="sub">
+        <h3>I'am a software developer</h3>
+        </sub-title>
+        <p class="p">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      </div>
+      <div class="mobile-img"><img src="./images/illustration-header-mobile.png" alt="Diagram illustration of works" class="headline-img"></div>       
+    </section>
+    <ul class="social first-icon">
+      <li class="git fico">
+        <a href="#"><img src="./images/Icon -GitHub.png" alt="GitHub logo"></a>
+      </li>
+      <li class="lin fico"><a href="#"><img src="./images/Icon - Linkedin.png" alt="Linkedin logo"></a></li>
+      <li class="ter fico"><a href="#"><img src="./images/Icon - Angelist.png" alt="Angelist logo"></a></li>
+      <li class="twitter fico"><a href="#"><img src="./images/Icon - Twitter.png" alt="Twitter logo"></a></li>
+      <li class="mess fico"><a href="#"><img src="./images/Icon - Medium.png" alt=" M logo"></a></li>
+    </ul> 
+    <!-- <div class="desktop-img"><img src="./images/Illustration-header@2x.png" alt="Diagram illustration 2"></div> -->
```

Comment:
- [ OPTIONAL] I would advise you to remove all unused commented codes from this file. Removing unused commented code makes your code look tidy. 👍